lib/matplotlib/tests/test_axes.py

```diff
@@ -6614,6 +6614,37 @@ def test_zoom_inset():
         axin1.get_position().get_points(), xx, rtol=1e-4)
 
 
+@image_comparison(['inset_polar.png'], remove_text=True)
```

Comment:
```suggestion
@image_comparison(['inset_polar.png'], remove_text=True, style='mpl20')
```

But this will also require regenerating the test image.

Reply:
done